src/applications/disability-benefits/all-claims/pages/choosePtsdType.js

```diff
@@ -0,0 +1,53 @@
+import { ptsdTypeDescription, ptsdTypeHelp } from '../content/ptsdTypeInfo';
+
+import { disabilityNameTitle } from '../content/newPTSDFollowUp';
+
+export const uiSchema = {
+  'ui:title': disabilityNameTitle,
+  'ui:description': ptsdTypeDescription,
+  'view:selectablePtsdTypes': {
```

Comment:
Should selecting at least one PTSD type be required? If so, here's how we've dealt with that before: https://github.com/department-of-veterans-affairs/vets-website/blob/73c8c90be51c18c35c40590cba9150cdf521b317/src/applications/disability-benefits/526EZ/config/form.js#L309

Reply:
Actually a better example is https://github.com/department-of-veterans-affairs/vets-website/blob/73c8c90be51c18c35c40590cba9150cdf521b317/src/applications/disability-benefits/all-claims/pages/evidenceTypes.js#L28
There's a `validateBooleanGroup` validator included in forms-system: https://github.com/department-of-veterans-affairs/vets-website/blob/73c8c90be51c18c35c40590cba9150cdf521b317/src/applications/disability-benefits/all-claims/pages/evidenceTypes.js#L1

Reply:
We're holding on this validation at the moment, there needs to be a business decision made whether we require them to select at least one or give them the option to select none and fill out 781 as they see fit("everything" is supposed to be optional within this form due to the stressful nature of filling it out).